### DISCUSSION TITLE
TuringCourses/major_module/history_of_cs_ideas/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#93] [General]
# TuringCourses/major_module/history_of_cs_ideas/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_module/history_of_cs_ideas/

**Comment by NeilAlfred** [comment]
上完这门课了来写个评论（x

一学期啥都不用干，只需要组队做一个 PPT + 5 min 展示，写一篇 5000 词以内的某计算机方向的文献综述（中英文都行，如果其他课写过，可以直接共享作业x），workload 非常小；

分别由 8 位老师，每隔两周上一次课。前 7 次爱去不去，都是科普性质的讲座，没有点名；最后一次课是期末考试，全班 300 多个人，60 多个组轮流展示，必须全组到场，缺课 = 缺考 = 零分；

是一门上课、考试、乃至给分都非常佛系 / 随机的课。本人一整个学期在这门课上花了 3 个小时不到的时间，获得了 2 个学分，3.9 的绩点，我觉得不算亏，见仁见智；总的来说是一门为了凑模块课学分的水课；
